ci/forge/Dockerfile.forge

```diff
@@ -1,8 +1,10 @@
 # syntax=docker/dockerfile:1.3-labs
+# shellcheck disable=SC2148
```

Comment:
what does this `#shellcheck` thing do?

I think I disabled the shellcheck lint on Dockerfiles

Reply:
shellcheck fixes in https://github.com/ray-project/ray/pull/38229